gravityflow/gravityflow
# Enhancements and fixes for the reports block

## Description
This PR adds enhancements and fixes for [the reports block PR](https://github.com/gravityflow/gravityflowblocks/pull/5):

1. Add the `display_filter` attribute to show or hide the filter in the shortcode.
2. Change the shortcode to use AJAX to update the report, when clicking on the 'Filter' button.
3. Support multiple Reports shortcodes or blocks on the same page.

## Testing instructions
1. Hide the filter in the Reports shortcode, test with `[gravityflow page="reports" display_filter="false"]`
2. Adding multiple Reports shortcodes in the same page, and see if they work correctly. This also validates we now update the reports with AJAX.
3. Go to WP Dashboard and check Workflow Reports, make sure it is still working.

## Automated Test Enhancements
Add an acceptance test that validates Reports shortcode can come with or without the filter. The test also validates Reports can only display to admins by default.

## Documentation Changes?
Add new `display_filter` to https://docs.gravityflow.io/article/36-the-shortcode.

## Checklist:
